MobalMobal/MobalMobal/InputChargingPoint/InputChargingPointViewController.swift

```diff
@@ -0,0 +1,152 @@
+//
+//  InputChargingPointViewController.swift
+//  MobalMobal
+//
+//  Created by 송서영 on 2021/03/05.
+//
+
+import SnapKit
+import UIKit
+
+class InputChargingPointViewController: UIViewController {
+    // MARK: - UIComponents
+    private let chargingInputView: UIView = {
+        let view: UIView = UIView()
+        view.layer.cornerRadius = 30
+        view.backgroundColor = UIColor(cgColor: CGColor(red: 255.0 / 255.0, green: 255.0 / 255.0, blue: 255.0 / 255.0, alpha: 0.07))
+        return view
+    }()
+    private let chargingInputField: UITextField = {
+        let textField: UITextField = UITextField()
+        textField.font = UIFont(name: "SpoqaHanSansNeo-Bold", size: 15)
+        textField.keyboardType = .numberPad
+        textField.attributedPlaceholder = NSAttributedString(string: "충전할 금액을 입력하세요", attributes: [NSAttributedString.Key.foregroundColor: UIColor.brownGreyTwo])
+        textField.textColor = .white
+        textField.addTarget(self, action: #selector(textEdited), for: .editingChanged)
+        return textField
+    }()
+    private let chargingViewImage: UIImageView = {
+        let imageView: UIImageView = UIImageView()
+        imageView.image = UIImage(named: "iconlyBrokenBuy")
+        imageView.layer.shadowRadius = 1.7
+        imageView.layer.shadowColor = UIColor.lightBluishGreen.cgColor
+        return imageView
+    }()
+    private let chargingButton: UIButton = {
+        let button: UIButton = UIButton()
+        button.layer.cornerRadius = 30
+        button.backgroundColor = UIColor(cgColor: CGColor(red: 71.0 / 255.0, green: 71.0 / 255.0, blue: 71.0 / 255.0, alpha: 1))
+        button.setTitle("충전하기", for: .normal)
+        button.setTitleColor(.brownGreyTwo, for: .normal)
+        return button
+    }()
+    let numberFormat: (Int) -> String = { number in
+        let str: String = "\(number)"
+        let regex: NSRegularExpression?
+        do {
+            regex = try? NSRegularExpression(pattern: "(?<=\\d)(?=(?:\\d{3})+(?!\\d))", options: [])
+        }
+        guard let regexString = regex else { return "" }
+        return regexString.stringByReplacingMatches(in: str,
+                                                    options: [],
+                                                    range: NSMakeRange(0, str.count),
```

Comment:
NSMakeRange()는 레거시 생성자입니다!!
```suggestion
                                                    range: NSRange(location: 0, length: str.count),
```